src/index.ts

```diff
@@ -157,7 +218,7 @@ const requestAndConfirmAirdrop = async (
   publicKey: PublicKey,
   amount: number,
 ) => {
-  let airdropTransactionSignature = await connection.requestAirdrop(
+  const airdropTransactionSignature = await connection.requestAirdrop(
```

Comment:
Good spot!

Reply:
Everything else looks good, get the above sorted and we can get this in today! 🙏